internal/artifacts/module_arch.go

```diff
@@ -464,12 +464,18 @@ func archiveModuleToResultDir(buildResult string, requestedResultFileName string
 }
 
 // getIgnores - get files and/or subfolders to exclude from the package.
-func getIgnores(module *mta.Module) []string {
+func getIgnores(moduleLoc dir.ITargetModule, module *mta.Module) []string {
 	var ignoreList []string
 	// ignore defined in build params is declared
 	if module.BuildParams != nil && module.BuildParams[ignore] != nil {
 		ignoreList = convert(module.BuildParams[ignore].([]interface{}))
 	}
+	// we add temporary folder to the list of ignores, because
+	// for the case when source == target and module path == project path,
+	// temporary folder can become part of the packaged module content
+	tmpFolder := filepath.Dir(moduleLoc.GetTargetModuleDir(module.Name))
+	tmpFolderName := filepath.Base(tmpFolder)
```

Comment:
Please use `filepath.Rel` to get the relative path. Note that if the temp folder is not under the module path it doesn't have to be ignored because it will not be packaged (not sure if adding it to the ignored list will cause an issue or not - please test this).